contracts/src/v0.8/ccip/pools/GHO/UpgradeableBurnMintTokenPool.sol

```diff
@@ -18,8 +19,14 @@ import {IRouter} from "../../interfaces/IRouter.sol";
 /// - Implementation of Initializable to allow upgrades
 /// - Move of allowlist and router definition to initialization stage
 contract UpgradeableBurnMintTokenPool is Initializable, UpgradeableBurnMintTokenPoolAbstract, ITypeAndVersion {
```

Comment:
@CheyenneAtapour can we add a docs saying we introduced the RateLimitAdmin capability?

Reply:
done